webapp/WEB-INF/templates/admin/system/manage_files_system.html

```diff
@@ -14,7 +14,7 @@
 						<td>${file.name}</td>
 						<td>${file.description}</td>
 						<td>
-							<@aButton href='jsp/admin/system/ManageFilesSytemDir.jsp?dir=${file.directory}' buttonIcon='eye' title='#i18n{portal.system.files_system.view} ${file.name}' showTitle=false />
+							<@aButton href='jsp/admin/system/ManageFilesSytemDir.jsp?dir=${file.directory}' buttonIcon='eye' title='#i18n{portal.system.manage_files_system.vView} ${file.name}' showTitle=false />

```

Comment:
Fixed in 431fd2fffabfab9419daa42358d8334646b084e6 and 21a847bdc5361929a3d5b603b10a878fd37b14cb